photon-api/src/main/scala/com/linkedin/photon/ml/optimization/RandomEffectOptimizationTracker.scala

```diff
@@ -89,57 +97,56 @@ protected[ml] class RandomEffectOptimizationTracker(val optimizationStatesTracke
    *
    * @return A map of convergence reason to its count of occurrence from the random effect optimization problems
    */
-  def countConvergenceReasons: Map[String, Int] = {
-    optimizationStatesTrackers.map { optimizationStatesTracker =>
-      if (optimizationStatesTracker.convergenceReason.isDefined) {
-        (optimizationStatesTracker.convergenceReason.get.reason, 1)
-      } else {
-        (RandomEffectOptimizationTracker.NOT_CONVERGED, 1)
-      }
-    }.reduceByKey(_ + _).collectAsMap()
-  }
+  def countConvergenceReasons: Map[String, Int] = optimizationStatesTrackers.map { optimizationStatesTracker =>
```

Comment:
Can we please add a carriage return between the signature and the body of the function for a multiline body? This makes it harder to read the return type of the function for me (esp. because the body doesn't even start with a `{`). And it is not consistent with `def getRegularizationTermValue(modelsRDD` below.